util/cmd.py

```diff
@@ -51,7 +51,7 @@ def common_args(parser, arglist=(('tmp_dir', None), ('loglevel', None))):
     for k, v in arglist:
         if k == 'loglevel':
             if not v:
-                v = 'DEBUG'
+                v = 'WARNING'
```

Comment:
why did this change?

Reply:
It was a bit loud - I'm not sure `DEBUG` was intentional or not.